storey/redis_driver.py

```diff
@@ -0,0 +1,482 @@
+import json
+import math
+import os
+import time
+from datetime import datetime, timedelta, timezone
+from enum import Enum
+from typing import List, Union, Optional
+import pandas as pd
+import asyncio
+import concurrent
+from functools import partial
+from .drivers import Driver
+
+import redis
+import rediscluster
+from .dtypes import RedisError
+from .utils import schema_file_name
+
+
+class RedisType(Enum):
+    STANDALONE = 1
+    CLUSTER = 2
+
+
+class NeedsRedisAccess:
+    """Checks that params for access to Redis exist and are legal
+
+    :param webapi: URL to the web API (https or http). If not set, the REDIS_URL environment variable will be used.
```

Comment:
I don't think this comment is correct for redis. Redis URLs have the "redis" scheme rather than "http" or "https", and are not referred to as "web API". So this seems like accidental c&p from v3io.

Reply:
fixed